elsie/render/backends/svg/backend.py

```diff
@@ -11,6 +12,8 @@
 from .query import compute_query
 from .rcontext import SvgRenderingContext
 
+VERSION_REGEX = re.compile(r"Inkscape (\d+\.\d+\.\d+)")
```

Comment:
This regexp is too strict. As we only need major version I would suggest to check it like:
Inkscape (\d+\..*)
It would be also more robust to a (potential) versions with suffix -dev/-test etc.